test/integration/serverless/snapshots/log-csharp

```diff
@@ -1,4 +1,17 @@
 [
+  {
+    "message": {
+      "message": "TIMESTAMP UTC | DD_EXTENSION | ERROR | Error happened when loading configuration from datadog.yaml for metric agent.\n",
```

Comment:
Alternatively, we could just add a `datadog.yaml` file for each of these tests. 😃 Might be worth doing on at least one of them since it could give us added test coverage.